test/cloud_signup_test.go

```diff
@@ -1,79 +0,0 @@
-package test
```

Comment:
I don't want a browser window popping up every time the unit tests run, and other than that there's basically nothing here that really seemed worth testing so I just removed this file completely.